modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt

```diff
@@ -1028,29 +1027,37 @@ class InstructionsTest : IrohaTest<Iroha2Client>() {
     @Story("Account transfers domain ownership")
     @SdkTestId("transfer_domain_ownership")
     fun `transfer domain ownership`(): Unit = runBlocking {
-        val genesisAccountId = AccountId(GENESIS.asDomainId(), GENESIS.asName())
-        var domain = QueryBuilder.findDomainById(DEFAULT_DOMAIN_ID)
+        val domainId = "Kingdom".asDomainId()
+
+        client.sendTransaction {
+            account(super.account)
+            registerDomain(domainId)
+            buildSigned(super.keyPair)
+        }.also { d ->
+            withTimeout(txTimeout) { d.await() }
+        }
+        val aliceDomain = QueryBuilder.findDomainById(domainId)
```

Comment:
to avoid confusion I'd suggest renaming the val to `newDomain` or `kingdomDomain`